BangazonAPI/BangazonAPI/Controllers/OrderController.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.Collections.Generic;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using BangazonAPI.Models;
+using Microsoft.AspNetCore.Http;
+
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrderController : ControllerBase
+    {
+
+        private readonly IConfiguration _config;
+
+        public OrderController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+
+        //Get method with two strings, 'include', 'completed'
+        [HttpGet]
+        public async Task<IActionResult> Get(string include, string completed)
+        {
+            using (SqlConnection conn = Connection)
```

Comment:
I can't test `?include=products`  because the seeder data you gave me from your database doesn't have any `OrderProducts` entries. Either add that to your seeder script or provide a mini-script in your PR testing instructions.